src/helpers.ts

```diff
@@ -508,11 +509,11 @@ export function getCurrentGitTag(): string {
  * Encodes a buffer as a `0x` prefixed lower-case hex string.
  * Returns an empty string if the buffer is zero length.
  */
-export function bufferToHexPrefixString(buff: Buffer): string {
+export function bufferToHexPrefixString(buff: Uint8Array): string {
   if (buff.length === 0) {
     return '';
   }
-  return '0x' + buff.toString('hex');
+  return '0x' + bytesToHex(buff);
```

Comment:
This function is called in a few hot paths, we should keep the node.js `Buffer` usage unless comprehensive benchmarking can show that this `byteToHex` function doesn't result in any regressions to CPU or memory usage.

Reply:
Got it. Is it ok to keep `Uint8Array` here, or should the underlying data also stay in `Buffer`?

Reply:
Needs to be `Buffer` since that is what has the `.toString('hex')` capability

Reply:
I mean, would wrapping the input Uint8Array in a `Buffer.from(buff)` work? Is the concern general regressions (from switching types) or specifically `bytesToHex`?

I think I'll revert the change for now and maybe keep two versions of stacks.js — otherwise some vars will have to be wrapped with either `Uint8Array.from` or `Buffer.from`.

Reply:
> would wrapping the input Uint8Array in a Buffer.from(buff) work?

Can you do that in the calling code? Most of the API codebase is using `Buffer` instances, and we don't want to incur any unnecessary memory or CPU penalties from constructing new `Buffer` instances and/or using `bytesToHex` rather than `Buffer.toString('hex')`.

I don't think you'd need `Uint8Array.from` anywhere, because `Buffer` is an instance of `Uint8Array`.

Reply:
Yep, sounds good.